lib/Db/Mailbox.php

```diff
@@ -138,6 +139,13 @@ public function hasLocks(int $now): bool {
 		return false;
 	}
 
+	/**
+	 * @return FolderStats
+	 */
+	public function getFolderStats(): FolderStats {
```

Comment:
nitpick: I wonder if we should rename `FolderStats` to `MailboxStats` and call this method simply `getStats` for clarification

we generally tried to unify terminology from *folder* to *mailbox* as that aligns better with Horde and the RFCs and therefore shall help lower potential for confusion

Reply:
I highly agree. Wasn't sure if the refactor is appropriate here but I can implement it.

Reply:
I'll allow it :wink:

Reply:
Done